src/backend/gl/src/device.rs

```diff
@@ -213,6 +213,32 @@ impl Device {
             })
     }
 
+    fn specialize_ast(
+        &self,
+        ast: &mut spirv::Ast<glsl::Target>,
+        specializations: &[pso::Specialization],
+    ) {
+        let spec_cons = ast.get_specialization_constants().unwrap();
+        for spec in specializations {
```

Comment:
I guess this part could be the same as [the DX12 logic](https://github.com/gfx-rs/gfx/blob/master/src/backend/dx12/src/device.rs#L396) as well